client/state/gutenberg/test/reducer.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
Remove this :D Let's use jest. No need to import chai.

Reply:
wasn't sure exactly what we were using, so I defaulted to chai. Fixed on e2a414c52abe956310f37e9028427af57bd6edad.